Tests/TuistCoreIntegrationTests/RootDirectoryLocatorIntegrationTests.swift

```diff
@@ -154,4 +154,24 @@ final class RootDirectoryLocatorIntegrationTests: TuistTestCase {
         // Then
         XCTAssertEqual(got, temporaryDirectory)
     }
+
+    func test_locate_when_multiple_workspaces_exist_but_git_can_return_top_level() throws {
+        // Given
+        let temporaryDirectory = try AbsolutePath(validating: "/private")
+            .appending(temporaryPath().relative(to: "/"))
```

Comment:
`/var` symlinks to `/private/var` on macOS. Temporary directory is made in `/var`, but Git will return absolute path from `/private/var`. So adjusting here seems like a simple solution